PWGDQ/Core/HistogramsLibrary.cxx

```diff
@@ -269,6 +269,9 @@ void o2::aod::dqhistograms::DefineHistograms(HistogramManager* hm, const char* h
       const int kPinEleNbins = 15;
       double pinEleBinLims[kPinEleNbins + 1] = {0.6, 0.8, 1.0, 1.2, 1.5, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 6.0, 8.0, 10.0, 15.0};
```

Comment:
Hi @zjxiongOvO , can you extend pin range down to 0.1 GeV/c for PID post-calibration? This is necessary for low mass dielectrons. e.g. pin = [0.1, 0.2, 0.4, 0.6 XXX]

Reply:
> Hi @zjxiongOvO , can you extend pin range down to 0.1 GeV/c for PID post-calibration? This is necessary for low mass dielectrons. e.g. pin = [0.1, 0.2, 0.4, 0.6 XXX]

Thank you for your comments. I will attempt to expand the momentum range to 0.1 GeV/c, but please note that it may result in increased CPU processing time.

Reply:
Thank you for implementing this change.